docs/FAQ.md

```diff
@@ -1,5 +1,15 @@
 # Frequently asked questions
 
+### Where are my "saves"? I thought `jj` saved the working copy after every edit.
+
+There is a working-copy commit that is updated on every jj command, and it so 
+happens that every commit has an associated change id, so it's true that there 
+is a "change" that's "caused" by running any jj command. You can observe this 
+by looking at the change id, the second highlighted part in `jj log` for a 
+commit, which changes after editing a file or the commit description.
+
+Use `jj interdiff` to see a commit's evolution,
```

Comment:
`jj obslog`? And it abruptly stops after comma :)

Reply:
done, as I've taken Ilya's suggestion from https://github.com/martinvonz/jj/pull/2459#discussion_r1375362392